scraper.rb

```diff
@@ -21,11 +21,20 @@ def scrape(h)
 ScraperWiki.sqliteexecute('DELETE FROM data') rescue nil
 
 start = 'http://sobranie.mk/current-structure-2014-2018.nspx'
-data = scrape(start => GroupsPage).groups.flat_map do |group|
+term = 2014
+
+groups_page = scrape(start => GroupsPage)
+
+current_members = groups_page.groups.flat_map do |group|
   scrape(group.source => GroupPage).members.map do |mem|
-    mem.to_h.merge(scrape(mem.source => MemberPage).to_h.merge(party: group.name, term: 2014))
+    mem.to_h.merge(scrape(mem.source => MemberPage).to_h.merge(party: group.name, term: term))
   end
 end
-# puts data.map { |r| r.reject { |k, v| v.to_s.empty? }.sort_by { |k,v| k }.to_h }
 
+ceased_members = scrape(groups_page.ceased_members_url => CeasedMembersPage).members.map do |mem|
+  mem.to_h.merge(scrape(mem.source => MemberPage).to_h.merge(party: mem.party, term: term))
+end
+
+# puts data.map { |r| r.reject { |k, v| v.to_s.empty? }.sort_by { |k,v| k }.to_h }
+data = current_members + ceased_members
```

Comment:
the commented out line is referring to a variable that doesn't exist yet. These should really be the other way around.